index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portifolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body> <!-- Header start-->
+    <header>
+        <div class="nav-container">
+            <a href="#" class="logo">WELCOME</a>
+            <nav>
+                <button class="menu" type="button" >
+                    <img src="./images/menu.svg" alt="a menu link">
+                </button>
```

Comment:
- [OPTIONAL] Great job using semantic HMTL5 tags 💯 I just suggest decreasing the size of  `menu` to match your Figma design 👍

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/198814469-198dce93-51b3-4ed4-867b-dd53e5fe86be.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/198814490-487ff6d9-cfea-46da-a05c-59a526c96554.png" width="500" /> |